lib/generators/clearance/specs/templates/features/clearance/visitor_resets_password_spec.rb.tt

```diff
@@ -3,6 +3,15 @@ require "support/features/clearance_helpers"
 
 RSpec.feature "Visitor resets password" do
   before { ActionMailer::Base.deliveries.clear }
+<% if defined?(ActiveJob) -%>
+
+  around do |example|
+    original_adapter = ActiveJob::Base.queue_adapter
+    ActiveJob::Base.queue_adapter = :inline
+    example.run
+    ActiveJob::Base.queue_adapter = original_adapter
```

Comment:
I actually don't know how this works. If the test raises an exception or fails, does that get caught within `example.run` or higher up the stack? I was assuming that `around` errors bubbled up the same way `before` errors did, and that it would be reasonable to expect `example.run` to raise an error here.